traptor/settings.py

```diff
@@ -51,6 +51,11 @@
 API_PORT = os.getenv('API_PORT', 5000)
 API_DIR = os.getenv('API_DIR', '.')
 API_SPEC = os.getenv('API_SPEC', 'spec.yml')
+API_BACKEND = os.getenv('API_BACKEND', 'piscina')
```

Comment:
can we default this to 'local' for people who normally just want to use the direct twitter api?